teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorKeysOptions.java

```diff
@@ -80,6 +81,35 @@
       arity = "1")
   private int validatorExternalSignerTimeout = 1000;
 
+  @CommandLine.Option(
+      names = {"--validators-external-signer-keystore"},
+      paramLabel = "<FILE>",
+      description =
+          "Keystore (PKCS12/JKS) to use for TLS mutual authentication with external signer",
+      arity = "1")
+  private String validatorExternalSignerKeystore = null;
+
+  @CommandLine.Option(
+      names = {"--validators-external-signer-keystore-password-file"},
+      paramLabel = "<FILE>",
+      description = "Password file for keystore",
```

Comment:
We should make this description specific to this type of keystore rather than having the same description as `validators-external-signer-truststore-password-file`

Reply:
Updated the description to be more descriptive.